### ISSUE TITLE
Last Minute Office Hours

### ISSUE THREAD

**Issue by afgong** [jmbejara/comp-econ-sp18#78]
Hey Jeremy! Would you be available to host last minute office hours late tomorrow morning? A few of us have quite a few questions, and we would love to talk through them with you as final preparation. Thanks so much!

**Comment by jmbejara** [comment]
Yep! Definitely! I just sent out the email. I will be holding office hours tomorrow morning at 11am until noon in the graduate lounge in Saieh.
